ToolChain/Assembler/Parser/Parser.py

```diff
@@ -573,3 +546,128 @@ def _generateBinaryInstructionFromInstruction(self, instruction: Instruction=Non
                                                                               memoryRefSymbol.encode())
 
         return binaryInstructionByteString
+
+    def translateRegisterNameToRegisterCode(self, registerName: str=""):
+        """
+        This takes a register name and returns a register code as per:
+            A = 0b00
+            B = 0b01
+            C = 0b10
+            ...
+            S = 0b111
```

Comment:
to S2

Reply:
Good point! Though @pg-cse suggested I try to keep comments as "future proof" as possible, so it's not impossible for future implementations to have more than just A-S2 registers. I may simply change the wording on this to make it more generic.